### DISCUSSION TITLE
Academic survey on the use of GenAI tools in UK software development -- if you are working in the UK please consider participating!

### DISCUSSION THREAD

**Discussion by Lena2019** [community/community#131136] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear UK software developers,

If you do not mind sparing 10-15 minutes from your time to participate in this survey, I would be extremely grateful!
[redacted by moderator]

The survey is about the use of GenAI tools from UK software developers, and your input will be extremely valuable.  ****As a token of your time, you will have a chance to enter into a draw to win one of five £30 Amazon gift cards**. You will also receive a **summary report** of the research results. 

Data will be anonymized and used for academic purposes only.

Thank you very much in advance for considering this request,

Kind regards,

Eleni Lioliou and Silvia Wu

**Comment by ebndev** [comment]
Hi @Lena2019, thanks for sharing with the community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
